internal/database/security_event_logs.go

```diff
@@ -45,6 +45,12 @@ const (
 	SecurityEventAccessTokenCreated     SecurityEventName = "AccessTokenCreated"
 	SecurityEventAccessTokenDeleted     SecurityEventName = "AccessTokenDeleted"
 	SecurityEventAccessTokenHardDeleted SecurityEventName = "AccessTokenHardDeleted"
+
+	SecurityEventGithubAuthSucceeded SecurityEventName = "GithubAuthSucceeded"
+	SecurityEventGithubAuthFailed    SecurityEventName = "GithubAuthFailed"
+
+	SecurityEventGitlabAuthSucceeded SecurityEventName = "GitlabAuthSucceeded"
+	SecurityEventGitlabAuthFailed    SecurityEventName = "GitlabAuthFailed"
```

Comment:
```suggestion
	SecurityEventGitHubAuthSucceeded SecurityEventName = "GitHubAuthSucceeded"
	SecurityEventGitHubAuthFailed    SecurityEventName = "GitHubAuthFailed"

	SecurityEventGitLabAuthSucceeded SecurityEventName = "GitLabAuthSucceeded"
	SecurityEventGitLabAuthFailed    SecurityEventName = "GitLabAuthFailed"
```

Sorry this is what happens when you tag Joe in a review 😛 and these are not nits.

Reply:
Nice, love it!